src/main/java/net/coderbot/iris/shaderpack/IdMap.java

```diff
@@ -59,20 +60,47 @@
 		// TODO: Properly override block render layers
 	}
 
+	/**
+	 * Loads properties from a properties file in a shaderpack path
+	 */
 	private static Optional<Properties> loadProperties(Path shaderPath, String name) {
-		Properties properties = new Properties();
+		if (shaderPath == null) return Optional.empty();
+
+		String fileContents = readProperties(shaderPath, name);
+		if (fileContents == null) return Optional.empty();
 
+		List<String> lines = parseProperties(name, fileContents);
+
+		StringBuilder processed = new StringBuilder();
+		for (String line : lines) {
+			processed.append(line);
+			processed.append('\n');
+		}
+
+		StringReader propertiesReader = new StringReader(processed.toString());
+		Properties properties = new Properties();
 		try {
-			properties.load(Files.newInputStream(shaderPath.resolve(name)));
+			properties.load(propertiesReader);
 		} catch (IOException e) {
-			Iris.logger.debug("An " + name + " file was not found in the current shaderpack");
+			Iris.logger.error("Error loading " + name + " at " + shaderPath);
+			Iris.logger.catching(Level.ERROR, e);
 
 			return Optional.empty();
 		}
 
 		return Optional.of(properties);
 	}
 
+	private static String readProperties(Path shaderPath, String name) {
+		try {
+			return new String(Files.readAllBytes(shaderPath.resolve(name)), StandardCharsets.UTF_8);
+		} catch (IOException e) {
```

Comment:
It's possible that this will be something other than a `NoSuchFileException`, that should be handled separately